syntax/requirements.vim

```diff
@@ -30,6 +30,12 @@ endif
 
 syn case match
 
+# https://pip.pypa.io/en/stable/reference/requirements-file-format/
+" https://pip.pypa.io/en/stable/reference/inspect-report/#example
```

Comment:
Are both comment lines?

Reply:
Fix '#' to '"'. These are some URL about requirements.txt format which can be referenced.